cmd/config/config.go

```diff
@@ -91,7 +91,7 @@ func main() {
 			usage(fs)
 			os.Exit(3)
 		}
-		if data, err = versions.Manager.Deploy(context.Background(), data, version); err != nil {
+		if data, err = versions.Manager.Deploy(context.Background(), data, uint16(version)); err != nil {
```

Comment:
Not in this diff, but you can rid the `else if version < 0` now since it's not possible

Reply:
Yep, thanks. Actually makes the earlier thing not so crazy.

Fixed https://github.com/gbjk/gocryptotrader/commit/cf86eeb468